src/Plaid/Entity/Institution.cs

```diff
@@ -23,28 +26,34 @@ public class Institution
 		/// A list of the Plaid products supported by the institution.
 		/// </summary>
 		[JsonPropertyName("products")]
-		public string[] Products { get; init; } = null!;
-
-		/// <summary>
-		/// A list of routing numbers associated with the institution.
-		/// </summary>
-		[JsonPropertyName("routing_numbers")]
-		public string[]? RoutingNumbers { get; init; }
+		public Product[] Products { get; init; } = null!;
 
 		/// <summary>
 		/// A list of country codes associated with the institution.
 		/// </summary>
 		[JsonPropertyName("country_codes")]
-		public string[]? CountryCodes { get; init; }
+		public string[] CountryCodes { get; init; } = null!;
```

Comment:
Is there a reason this field is `string[]` with an initializer `= null!` where in `InstitutionSearchOptions` you are using `string[]?`? Maybe it's a struct vs class thing but just curious.

Reply:
Different spec from Plaid. Here, they promise to always provide it. In some instances of `InstitutionSearchOptions`, they do not require that it is provided. Others do require it is provided, and it is `string[]` there as well. 

I'm looking forward to C#10 where I can just mark it `required` and not do this `= null!` stuff. :)